federation/pkg/federation-controller/util/eventsink/eventsink.go

```diff
@@ -48,10 +49,11 @@ func NewFederatedEventSink(clientset fedclientset.Interface) *FederatedEventSink
 var scheme = runtime.NewScheme()
 
 func init() {
-	scheme.AddKnownTypes(clientv1.SchemeGroupVersion,
-		&clientv1.Event{},
-		&kubev1.Event{},
-	)
+	// register client-go's and kube's Event type under two different GroupVersions
+	// TODO: switch to client-go client for events
+	scheme.AddKnownTypes(clientv1.SchemeGroupVersion, &clientv1.Event{})
+	scheme.AddKnownTypes(schema.GroupVersion{Group: "kube-" + kubev1.SchemeGroupVersion.Group, Version: kubev1.SchemeGroupVersion.Version}, &kubev1.Event{})
```

Comment:
Oh, I remember this.  It was for my conversion shim.  Yeah. just make the name clearly bad and we can live with it.